eng/Versions.props

```diff
@@ -147,6 +151,12 @@
     <SystemIdentityModelTokensJwtPackageVersion>5.3.0</SystemIdentityModelTokensJwtPackageVersion>
     <WindowsAzureStoragePackageVersion>8.1.4</WindowsAzureStoragePackageVersion>
 
+    <!-- Dependencies for Blazor. -->
+    <MicrosoftAspNetCoreBlazorMonoPackageVersion>0.8.0-preview1-20181126.1</MicrosoftAspNetCoreBlazorMonoPackageVersion>
+    <!-- When updating this, ensure you also update src/Components/Browser.JS/src/package.json to reference the corresponding version of @dotnet/jsinterop -->
+    <MonoWebAssemblyInteropPackageVersion>0.8.0-preview1-20181126.4</MonoWebAssemblyInteropPackageVersion>
+    <MicrosoftJSInteropPackageVersion>0.8.0-preview1-20181126.4</MicrosoftJSInteropPackageVersion>
```

Comment:
@pranavkm do you anticipate any work will be needed when updating to the latest versions coming out of aspnet/Extensions? In this PR, I'm only trying to change how we reference things, not what we reference. 

To make this version flow automatically, we need to add a maestro/darc subscription.